### PR TITLE
Added string subsequence in JS

### PR DESCRIPTION
**Pull request by toihirhalim**
add sequence.js
and link to readme.md

### CODE REVIEW

File: strings/js/sequence.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        aaa [ 'aaa', 'aa', 'a' ]
+        hello [
+            'hello', 'hell', 'helo', 'hel',
+            'heo',   'he',   'hllo', 'hll',
+            'hlo',   'hl',   'ho',   'h',
+            'ello',  'ell',  'elo',  'el',
+            'eo',    'e',    'llo',  'll',
+            'lo',    'l',    'o'
+        ]
+*/
```
Comment (cclauss) [inline strings/js/sequence.js]:
```suggestion
*/

```
Files should end with one and only one carriage return (\n) so that GitHub gets rid of that ⛔

**Comment by toihirhalim** [comment]
hi ***@***.****
I'm not sure i understand what you mean  here
can you be a little more specific ?
thanks

**Comment by cclauss** [comment]
Your `remove extra line` commit fixed the problem.

**Comment by toihirhalim** [comment]
oh great

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**